convert2rhel/main.py

```diff
@@ -113,22 +114,17 @@ def main_locked():
         # actions.run_actions() (either from a bug or from the user hitting
         # Ctrl-C)
         process_phase = ConversionPhase.PRE_PONR_CHANGES
-        pre_conversion_results = actions.run_actions()
+        pre_conversion_results = actions.run_pre_actions()
 
         if toolopts.tool_opts.activity == "analysis":
             process_phase = ConversionPhase.ANALYZE_EXIT
             raise _AnalyzeExit()
 
-        pre_conversion_failures = actions.find_actions_of_severity(
-            pre_conversion_results, "SKIP", level_for_raw_action_data
-        )
-        if pre_conversion_failures:
-            # The report will be handled in the error handler, after rollback.
-            loggerinst.critical("Conversion failed.")
+        _raise_for_skipped_failures(pre_conversion_results)
 
         # Print the assessment just before we ask the user whether to continue past the PONR
         report.summary(
-            pre_conversion_results,
+            results=pre_conversion_results,
```

Comment:
I will add this into my PR #1153 to have the call consistent across the `main.py`